Docs/user_guide/getting_started.md

```diff
@@ -123,6 +123,15 @@ The included libcrypto.so.1.1 in the Slicer installation is incompatible with th
 
     $SLICER_ROOT/lib/Slicer-4.xx/libcrypto.*
 
+If you double-clicked on the app and it doesn't launch, you can try this (in order):
+- verify execution rights of the app (Right click on the app > properties > executable as a program checked) or in terminal, in the app folder:
+    
+    chmod +x Slicer
```

Comment:
This shouldn't be needed since the file will be executable after extracting from the tar file.  Did you ever have it happen that this was not the case?

Reply:
Some unzip software have special usage. Like Info-Zip 3.0 for instance, where you need to unzip as root if you want to restore the rights scheme on the files. Some also configure their unzip software to restore files only in 666 (rw-rw-rw-) for security reasons. So you can't pass this option to troubleshoot i think (or at least you have to check that permissions are 755...)

Reply:
Thanks, that makes more sense.  Maybe you can make a small edit to the documentation text summarizing this.

Reply:
Sure i'll be happy to...

Reply:
Would you be interested in a french translation of the documentation ?

Reply:
> Would you be interested in a french translation of the documentation ?

Yes!  In fact, there's a project devoted to that and any help would be greatly appreciated!

There is an internationalization group and infrastructure that you can learn about here: 

https://projectweek.na-mic.org/PW39_2023_Montreal/Projects/3DSlicerInternationalization/

https://chanzuckerberg.com/eoss/proposals/3d-slicer-in-my-language-internationalization-and-usability-improvements/

The first fully-French tutorial was recently held, but there is a lot more to be done:

https://sliceriml.github.io/

@spujol is the PI of these projects.